src/components/ProductCard.test.tsx

```diff
@@ -36,17 +41,6 @@ jest.mock('@/components/ColorDots', () => {
     </div>
   );
 
-  ColorDots.propTypes = {
```

Comment:
Removed Proptypes, as from now on we can directly use types within the test files.